src/v2/guide/transitioning-state.md

```diff
@@ -519,15 +517,15 @@ new Vue({
 </script>
 {% endraw %}
 
-Within child components, we can use any combination of transition strategies that have been covered on this page, along with those offered by Vue's [built-in transition system](transitions.html). Together, there are very few limits to what can be accomplished.
+Dalam anak komponen, kita dapat menggunakan kombinasi strategi transisi apapun yang telah tercakup pada halaman ini, bersama dengan yang ditawarkan oleh [sistem transisi *built-in* Vue](transitions.html). Dengan demikian, batasan menjadi minim dari apa yang bisa diperoleh.
 
-## Bringing Designs to Life
+## Menghidupkan Desain
 
-To animate, by one definition, means to bring to life. Unfortunately, when designers create icons, logos, and mascots, they're usually delivered as images or static SVGs. So although GitHub's octocat, Twitter's bird, and many other logos resemble living creatures, they don't really seem alive.
+Menganimasi, oleh satu definisi, berarti menghidupkan. Sayang sekali, ketika desainer membuat ikon, logo, dan maskot, mereka biasanya menyerahkannya dalam bentuk gambar atau SVG statis. Jadi meskipun *octocat* Github, burung Twitter, dan banyak logo lain menyerupai makhluk hidup, mereka tidak terkesan sungguh-sungguh hidup.
```

Comment:
_By one definition_, ini sebuah istilah mas. yg mana menjelaskan bahwa sesuatu itu dapat diartikan banyak, contoh `nodejs, by one definition, are just chrome in a server` artinya bisa itu, atau bisa juga yg lain. Menurut saya lebih baik diartikan seperti ini

```
by one definition => dapat diartikan \\ by one definition => dapat didefinisikan 
```

jika digabung maka

```md
- Menganimasi dapat didefinisikan memberikan kehidupan
```

mungkin memang kita kehilangan frasa kata Inggris-nya tapi saya rasa tidak apa apa (Selagi masih nyambung ke kalimat berikutnya). Mohon dipertimbangkan.

Reply:
saya setuju dengan translasi tsb mas, terdengar lebih natural dalam bahasa Indonesia, tetapi apakah tidak perlu memakai kata `sebagai` setelahnya mas?
maksud saya seperti ini
`Menganimasi dapat didefinisikan sebagai memberi kehidupan`

Reply:
LGTM mas.